recipemenager.py

```diff
@@ -1,4 +1,4 @@
-class RecipeManager:
+class recipemanager:
```

Comment:
As per the standard (https://peps.python.org/pep-0008/#class-names) this was correct the 1st time around and capitalisation should be used.

Reply:
so just : class RecipeManager?